tools/automatic_query_fixer/src/main/java/com/google/cloud/bigquery/utils/queryfixer/QueryFixerMain.java

```diff
@@ -56,10 +59,10 @@ private static CommandLine readFlags(String[] args) {
     options.addOption(option);
     option =
         new Option(
-            /*opt=*/ INTERACT_SHORTCUT,
-            /*long-opt=*/ INTERACT,
+            /*opt=*/ MODE_SHORTCUT,
+            /*long-opt=*/ MODE,
             /*hasArg=*/ true,
-            /*description=*/ "Interactive Mode. The available mode are \"none\" (default), \"guide\" and \"all/full\"");
+            /*description=*/ "Interactive Mode. The available mode are \"auto\" (default), \"ua/user-assistance\" and \"fo/fix-once\". Please see the README file for their meanings");
```

Comment:
nit: "modes"

Reply:
nit: "their meanings" -> "the detailed description."

Reply:
Done